src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj

```diff
@@ -2,8 +2,68 @@
 
   <PropertyGroup>
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains MSBuild support for Razor.</Description>
-    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+
+    <!-- 
+      Using netcoreapp2.0 here is a workaround for https://github.com/Microsoft/msbuild/issues/2661
+
+      We need to use a P2P reference to build some other projects that need to target netcoreapp2.0
+      and even though we don't actually want to reference the output, MSBuild will not allow it.
+    -->
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
We don't have anything in lib. @pranavkm assured me that this would do the right thing.

I can't manually author the `.nuspec` because it depends on the publish output of the tools, which can include corefx assemblies, that would be really fragile.

I'm going to create an item for this and move on. My only concern right now is that this doesn't break our build system.

Reply:
Logged #1843 to track this.

Reply:
@natemcmaster - I checked and you're right! OK, we'll have to deal with this somehow.